fotoapparat/src/main/java/io/fotoapparat/hardware/v1/capabilities/CapabilitiesFactory.java

```diff
@@ -87,4 +91,25 @@ public Capabilities fromParameters(Camera.Parameters parameters) {
         return result;
     }
 
+    private Set<Range<Integer>> extractPreviewFpsRanges(Camera.Parameters parameters) {
```

Comment:
You can skip the test for `v2` since we are not actively support it, but it would certainly be nice to have them for `v1`.

You can take a look at `CapabilitiesFactoryTest` - there we are mocking `Parameters` class from Android to return what we need. Should be rather simple to add similar test for FPS ranges.